addons/web/static/src/js/views/form/form_renderer.js

```diff
@@ -1149,6 +1192,19 @@ var FormRenderer = BasicRenderer.extend({
     _onNotebookTabChanged: function () {
         core.bus.trigger('DOM_updated');
     },
+    /**
+     * @private
+     * @param {OdooEvent} ev
+     */
+    _onQuickEdit: function (ev) {
+        if (this.mode === 'readonly' &&
+            !this.quickEditionExclusion.some(x => ev.data.target.closest(x))
+        ) {
+            this.quickEditInfo = ev.data;
+        } else {
+            ev.stopPropagation();
+        }
+    },
```

Comment:
let me summarize the flow as I understand it:
 1) a field is clicked, it triggers 'quick_edit' with some info
 2) the event goes through the renderer which conditionally stops it, and if not, stores that info
 3) the event is catched by the controller which switches to edit mode, re-renders the view, and calls autofocus
 4) in autofocus, we detect that we come from a quick edit flow, and in this case we find the field and call executeQuickEdit

Wouldn't it be simpler to:
 1) a field is clicked, it triggers 'quick_edit' with some info
 2) the event goes up to the controller directly, which conditionally decides to ignore it (with the exclusion selectors), and if not, switches to edit more and **then** calls a specific method of the renderer with the event info
 3) in the renderer, that method executes the quick edit on the correct field

Note that I'm not sure to understand why we need that exclusion selector list in the renderer/controller